doc/Command_Index_internal.tex

```diff
@@ -585,10 +582,10 @@ \section{Gregorio\TeX{} Controls}
 Macro called before each glyph.
 
 \macroname{\textbackslash gre@normalinitial}{}{gregoriotex-main.tex}
-Macro called at the end of the score to ensure that a big initial setting doesn't carry into the next score.
+Macro called at the end of the score to ensure that a big initial setting doesn’t carry into the next score.
 
 \macroname{\textbackslash greoldcatcode}{}{gregoriotex.tex}
-Macro to store the catcode for ``\@'' so that we can use said symbol in function names under Plain\TeX\ and then restore the original catcode after the package is done loading.
+Macro to store the catcode for ``@‘’ so that we can use said symbol in function names under Plain\TeX\ and then restore the original catcode after the package is done loading.
```

Comment:
A number of sites seem to use that (capital P, with a space), so I think it's probably fine to leave it at that.  Technically, I think the lowest level of TeX we support is e-TeX, which is actually a bit of a superset over "Plain TeX".